bears/go/GoMetaLintBear.py

```diff
@@ -0,0 +1,83 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GoRequirement import GoRequirement
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.settings.Setting import typed_list
+
+
+@linter(executable='gometalinter',
+        output_format='regex',
+        output_regex=r'\w+\.go:'
```

Comment:
Invalid regex, the regex mentioned in their repo is `<file>:<line>:[<column>]: <message> (<linter>)`